rkt/pubkey/pubkey.go

```diff
@@ -42,6 +42,14 @@ type Manager struct {
 type AcceptOption int
 type OverrideOption int
 
+type ErrKeyForPrefixExists struct {
```

Comment:
why not an error variable instead of a type? (e.g. https://github.com/coreos/rkt/blob/722f2bd38b08d28a6a56810719bd6dce6ad828e4/pkg/lock/file.go#L24)